fastestimator/trace/trace.py

```diff
@@ -144,10 +144,8 @@ class Logger(Trace):
     Args:
         extra_log_keys (set): set of keys to print from system buffer
     """
-    def __init__(self, extra_log_keys: Set[str], loss_names: Set[str]):
-        super().__init__(inputs=extra_log_keys | loss_names)
-        self.extra_log_keys = extra_log_keys
-        self.loss_names = loss_names
+    def __init__(self, extra_log_keys: Set[str]):
+        super().__init__(inputs=extra_log_keys)
```

Comment:
what's the plan for keeping the loss keys in scope during training so they still get brought back from gpu to cpu?

Reply:
During training, loss keys are included in Logger's `extra_log_keys`

Reply:
ah ok